.github/workflows/matrix_test.yml

```diff
@@ -52,11 +52,16 @@ jobs:
         pip install -r setup/requirements/dev-requirements.txt
         pip install -e .
 
-    - name: Run Tests
+    - name: Run Tests with FireFox
```

Comment:
I would prefer to again remove the `with FireFox` part, since this stage runs all tests, not only the browser tests. It should be sufficient to have the precise name of `Run Browser Tests With Chrome` for the next stage.

Reply:
Changed